src/API/Bitbucket/BitbucketAPI.php

```diff
@@ -47,22 +47,33 @@ protected function apiClient()
 
     protected function isPagedResponse(ResponseInterface $res)
     {
-        return true;
+        $responseBody = json_decode($res->getBody(), true);
+        if (!empty($responseBody['next']) || ! empty($responseBody['previous'])) {
+            return true;
+        }
+        return false;
     }
 
     protected function getPagerInfo(ResponseInterface $res)
     {
-        return [];
+        $responseBody = json_decode($res->getBody(), true);
+        $pagerInfo = [];
+        foreach (['next', 'previous'] as $type) {
+            if (isset($responseBody[$type])) {
+                $pagerInfo[$type] = $responseBody[$type];
+            }
+        }
+        return $pagerInfo;
     }
 
     protected function isLastPage($page_link, $pager_info)
     {
-        return true;
+        return empty($pager_info['next']);
     }
 
     protected function getNextPageUri($pager_info)
     {
-        return null;
+        return !empty($pager_info['next']) ? $pager_info['next'] : false;
```

Comment:
In both the (GitHub)[https://github.com/pantheon-systems/terminus-build-tools-plugin/blob/master/src/API/GitHub/GitHubAPI.php#L121] and (GitLab)[https://github.com/pantheon-systems/terminus-build-tools-plugin/blob/master/src/API/GitLab/GitLabAPI.php#L139] implementations of `getNextPageUri()` we return `NULL` instead of `false`. I'm assuming PHP is evaluating both of them equivalently in our code but I'm thinking we should have this be consistent with the others.

Reply:
Sounds good, updated in 15d7482